app/src/main/AndroidManifest.xml

```diff
@@ -16,9 +16,8 @@
             android:value="@string/google_api_key" />
         <meta-data
             android:name="com.google.android.gms.version"
-            android:value="@integer/google_play_services_version"/>
-        <activity android:name=".MapActivity"
-            android:parentActivityName=".MainActivity"/>
+            android:value="@integer/google_play_services_version" />
+
```

Comment:
Is this file formatted? the blank space looks odd to me

Reply:
Yes it is formatted